tests_integration/__tests__/__snapshots__/piped-output.js.snap

```diff
@@ -12,6 +32,44 @@ exports[`no file diffs with --list-different + formatted file (write) 1`] = `Arr
 
 exports[`no file diffs with --list-different + formatted file (write) 2`] = `Array []`;
 
+exports[`output with --check + unformatted differs when piped (stderr) 1`] = `""`;
+
+exports[`output with --check + unformatted differs when piped (stderr) 2`] = `""`;
+
+exports[`output with --check + unformatted differs when piped (stdout) 1`] = `
+"Checking formatting...
+unformatted.jsunformatted.js
```

Comment:
One thing I don't quite understand is why `{ stdoutIsTTY: true }` does not produce a truly TTY behaviour (there are three cases similar to this). Here, the first instance of `unformatted.js` should have been deleted by `readline.clearLine(process.stdout, 0); readline.cursorTo(process.stdout, 0, null);`, but this does not happen. When using the same options in a macOS terminal, all works fine. 🤔

Reply:
Looks like the mocked stdout [only supports text being appended](https://github.com/kachkaev/prettier/blob/215ff43a4713878146936e30fa25cd25f3d54844/tests_integration/runPrettier.js#L32).

Reply:
OK, thanks! Looks like we can leave things as is then.